ro-base.obo

```diff
@@ -2519,6 +2533,7 @@ name: enabled by
 def: "inverse of enables" []
 subset: http://purl.obolibrary.org/obo/valid_for_gocam
 property_value: IAO:0000117 https://orcid.org/0000-0002-6601-2165
+property_value: seeAlso Enabled:by
```

Comment:
?

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)